docs/conceptual-guides/cookieplone-frontend-addon.md

```diff
@@ -0,0 +1,204 @@
+---
+myst:
+  html_meta:
+    "description": "An explanation on the anatomy of a frontend add-on"
+    "property=og:description": "An explanation on the anatomy of a frontend add-on"
+    "property=og:title": "Frontend add-on anatomy"
+    "keywords": "Seven, cookieplone, template, add-on, anatomy"
+---
+
+# Frontend add-on anatomy
+
+After you create a frontend add-on with the {term}`cookieplone` generator {doc}`../get-started/create-package`, the code lives in `packages/<add-on-name>`.
+
+## Root folder structure
+
+The generated project contains these relevant files and folders:
+
+`registry.config.ts`
+: This file is the main entry point for your add-on configuration.
+  You can programmatically register add-ons in your app.
+  By default, it registers your add-on and the official Seven add-ons.
+  The default Seven set includes the {term}`CMSUI` add-ons, but you can opt in to use only the {term}`PublicUI` add-ons when building a public site without CMS functionality.
```

Comment:
We went with a space between words for https://6.docs.plone.org/glossary.html#term-Public-UI

```suggestion
  The default Seven set includes the {term}`CMSUI` add-ons, but you can opt in to use only the {term}`Public UI` add-ons when building a public site without CMS functionality.
```